src/handlers/proposalVersionsHandlers.ts

```diff
@@ -145,47 +144,29 @@ const postProposalVersion = (
 						'proposalVersions.insertOne',
 						req.body,
 					);
-				logger.debug(proposalVersionQueryResult);
 				const proposalVersion = proposalVersionQueryResult.rows[0];
 				if (proposalVersion !== undefined) {
-					const proposalFieldValueQueries = req.body.fieldValues.map(
-						async (fieldValue) => {
-							const { value, applicationFormFieldId, position } = fieldValue;
+					const proposalFieldValues = await Promise.all(
+						fieldValues.map(async (fieldValue) => {
+							const { value, applicationFormFieldId } = fieldValue;
```

Comment:
I was like "wait, what happened to position?" but I see below it is still included in the `...fieldValue` and no longer needed as a  variable in this scope.